src/JustSaying/JustSayingBus.cs

```diff
@@ -194,7 +194,7 @@ private async Task RunImplAsync(CancellationToken stoppingToken)
             catch (OperationCanceledException)
             {
                 _log.LogDebug("Suppressed an exception of type {ExceptionType} which likely" +
```

Comment:
I think it would be clearer to add the space to the end of the previous sentence (or just put it on one line, the splitting it into two doesn't really seem necessary other than arbitrary length length).

Reply:
Done 👍